src/components/registration/registration-form/RegistrationForm.tsx

```diff
@@ -76,11 +82,15 @@ const RegistrationForm = ({
     parentDynamicFields,
   } = useContext(DynamicFieldsContext);
 
-  const [family, setFamily] = useState<FamilyFormData>(
-    existingFamily !== null
-      ? familyResponseToFamilyFormData(existingFamily)
-      : defaultFamilyData
-  );
+  const [enrolment, setEnrolment] = useState<EnrolmentFormData>({
+    family:
+      existingFamily !== null
+        ? familyResponseToFamilyFormData(existingFamily)
+        : defaultFamilyData,
+    preferred_class: null,
+    session: session.id,
+    status: EnrolmentStatus.REGISTERED,
```

Comment:
i went with registered because this should be the selected status most of the time (signed up is only for special cases)!

i'll merge this for now but double check with design